.env

```diff
@@ -1,15 +1,19 @@
-BACKEND_URL=https://helo-system.herokuapp.com/
-DISCORD_CLIENT_ID=
-DISCORD_CLIENT_SECRET=
-DISCORD_GUILD_ID=
-DISCORD_IGNORE_ROLES=
+# General
+BACKEND_URL=https://api.helo-system.de/
 DISCORD_INVITE=
-DISCORD_REPORT_MATCH_WEBHOOK=
-DISCORD_TEAMMANAGER_ID=
-NEXT_PUBLIC_FF_REPORT_MATCH_ALLOW_OFFENSIVE=
-NEXT_PUBLIC_FF_REPORT_MATCH_ALWAYS_ALLOW_OTHER=
+DISCORD_REPORT_MATCH_WEBHOOK=https://discord.com/api/webhooks/1007044837138251898/4BaIajLClvLu9S0CWA_GtdK-rvdn7BTx_0nZdANHAW7b78LIa9Hv_TRE1Z87m9ibee9G
+
+# Tracking
 NEXT_PUBLIC_MATOMO_JS=
 NEXT_PUBLIC_MATOMO_PHP=
 NEXT_PUBLIC_MATOMO_SITE_ID=
 NEXT_PUBLIC_MATOMO_URL=
+
+# Auth
 NEXTAUTH_SECRET=
+NEXT_PUBLIC_AUTH_SIGNIN_URL=http://localhost:5000/auth/discord/login
```

Comment:
these are all just values for local development, right? I assume you will add the correct values for the deployment after merge? 🤔